### PR TITLE
Don't double encode issue name

### LINKED ISSUE #664: Bug: API params got encoded twice
A few issue pages now have problem displaying the right set of data. This is because the params got encoded twice

e.g.,
- https://api.mozillapulse.org/entries/?issue=Digital%2520Inclusion
- should really be https://api.mozillapulse.org/entries/?issue=Digital%20Inclusion

I will submit a patch soon.

### PR DESCRIPTION
**Pull request by mmmavis**
Fixes #664

### CODE REVIEW

File: components/project-loader/project-loader.jsx [hunk 1]
```diff
@@ -67,6 +67,10 @@ export default React.createClass({
       delete combinedParams.bookmarkedOnly;
     }
 
+    if (combinedParams.showCounter) {
```
Comment (Pomax) [inline components/project-loader/project-loader.jsx]: do we need to delete though? the API should just ignore unknown args

File: pages/issue.jsx [hunk 2]
```diff
@@ -10,6 +10,6 @@ export default function (props) {
   return <div>
             <Helmet><title>{issueName}</title></Helmet>
             <IssueSelector />
-            <ProjectLoader issue={encodeURIComponent(issueName)} showCounter={true} />
+            <ProjectLoader issue={issueName} showCounter={true} />
```
Comment (Pomax) [inline pages/issue.jsx]: oh man, is there a way for us to add a test to the code that will catch things like this in the future?

File: components/project-loader/project-loader.jsx (hunk 1 above)
Comment (mmmavis) [inline components/project-loader/project-loader.jsx]: not really but i thought this would make it a cleaner url

File: pages/issue.jsx (hunk 2 above)
Comment (mmmavis) [inline pages/issue.jsx]: yea, i created a ticket: https://github.com/mozilla/network-pulse/issues/666

File: components/project-loader/project-loader.jsx (hunk 1 above)
Comment (Pomax) [inline components/project-loader/project-loader.jsx]: usually it's better to not call `delete`, so I'd say let's leave this out.

**Pull request merged**